src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -307,6 +323,22 @@ private void RefreshCondensedDisplay(Action refreshDisplay)
 
             string newContent = "null";
 
+            if (mirrorData is ProtoCore.MirrorDataInProgress
+                // When CBN is created, its cached value is set to MirrorDataInProgress.
+                // And it's not updated after update graph task is complited.
+                && !((nodeViewModel.NodeLogic is CodeBlockNodeModel) && (nodeViewModel.NodeLogic as CodeBlockNodeModel).Code == String.Empty))
```

Comment:
If it is only limited to code block nodes, the conditions are not written correctly. The 1st condition looks good enough.